### DISCUSSION TITLE
Entity Framework Net Core 5

### DISCUSSION THREAD

**Discussion by pacbel** [balta-io/7194#12] [Perguntas e Respostas] [answered]
Balta, entendi bem sobre o domínio e suas entidades. Você poderia fornecer um material ou indicar um que ensine o relacionamento entre os objetos (Key / ForeignKey) ,  IEntityTypeConfiguration para um banco CodeFirst?

**Comment by andrebaltieri** [comment, marked as answer]
Show!
https://docs.microsoft.com/pt-br/ef/core/modeling/relationships?tabs=fluent-api%2Cfluent-api-simple-key%2Csimple-key

**Reply by pacbel** [reply]
Muito obrigado! Meu nome é Carlos Pacheco minha empresa é Sistema Virtual Pacbel. Sou seu aluno Premium participo dos encontros como Sistema Virtual. Valeu pela dica :)

**Reply by andrebaltieri** [reply]
<3
